### ISSUE TITLE
cardinality is exported but undefined

### ISSUE THREAD

**Issue by daanhb** [JuliaApproximation/DomainSets.jl#90]
See #89

**Comment by daanhb** [comment]
@dlfivefifty what are the uses of the `cardinality` function? I only see it in [this line](https://github.com/JuliaApproximation/ContinuumArrays.jl/blob/6a4a8c07c12804d3dc89e017b654341125b56996/src/ContinuumArrays.jl#L31).
Currently, IntervalSets defines the `width` of an interval, and DomainSets defines the more general `volume` of a domain. I don't think we define any domains for which the cardinality would be finite, except perhaps vectors (seen as a domain) and a Point.

It may be interesting to define it, but does not seem essential to me. Does it have to be in DomainSets, or could it be defined in ContinuumArrays?

**Comment by dlfivefifty** [comment]
It can be in ContinuumArrays.jl. It's just used for the following:
```julia
julia> size(LinearSpline(range(0,1;length=10)))
(ℵ₁, 10)
```

**Comment by daanhb** [comment]
Okay, in that case I'll probably take it out here. I wonder if there are more exported-but-undefined symbols.

**Comment by daanhb** [comment]
Closed by #93

**Comment by daanhb** [comment]
It was the only one:
```julia
julia> using DomainSets

julia> filter(n -> ! isdefined(DomainSets,n), names(DomainSets))
1-element Vector{Symbol}:
 :cardinality
```
